index.html

```diff
@@ -587,6 +587,29 @@ <h3>DID Resolution Options</h3>
 				function and MUST NOT be used with the <code>resolve</code> function.
 			</dd>
 		</dl>
+		<dl>
+			<dt>versionId</dt>
+			<dd>
+				Identifies a specific version of a <a>DID document</a> to be resolved (the
+				version ID could be sequential, or a <a>UUID</a>, or method-specific).
+				If present, the associated value MUST be an <a
+					data-lt="ascii string">ASCII string</a>.
+			</dd>
+		</dl>
+		<dl>
+			<dt>versionTime</dt>
+			<dd>
+				Identifies a certain version timestamp of a <a>DID document</a> to be resolved.
+				That is, the <a>DID document</a> that was valid for a <a>DID</a> at a certain
+				time. If present, the associated value
```

Comment:
```suggestion
				That is, the most recent version of the <a>DID document</a> that was valid for a <a>DID</a> 
				before the specified `versionTime`. If present, the associated value
```

Something like this @ankurdotb @TallTed?

Note, I will need to update the definition in DID parameters to. But will do that once we have agreed on the change.

Reply:
Yes, this makes sense and helps disambiguate

Reply:
Looks good to me